client/containers/Balance/index.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+import {
+  getBalance,
+  getInflowBalance,
+  getOutflowBalance
+} from 'modules/transactions';
+
+import BalanceItem from 'components/Balance';
+import BalanceRow from 'components/Balance/BalanceRow';
+
+@connect(
+  state => ({
+    balance: getBalance(state),
+    inflow: getInflowBalance(state),
+    outflow: getOutflowBalance(state)    
+  })
+)
+class Balance extends Component {
+  static propTypes = {
+    balance: PropTypes.string.isRequired,
+    inflow: PropTypes.string.isRequired,
+    outflow: PropTypes.string.isRequired
+  }
+
+  render() {
+    const { inflow, outflow, balance } = this.props;
+
+    return (
+      <BalanceRow>
+        <BalanceItem amount={inflow} title="Total Inflow" />
+        <BalanceItem amount={outflow} title="Total Outflow" symbol="-" />
```

Comment:
I'd use a component for showing prefix (symbol). It feels more intuitive, but maybe that's just me.